test/integration/file_artifact_test.js

```diff
@@ -222,7 +222,7 @@ suite('artifact extration tests', () => {
       }
     });
 
-    let errorMessage = '[taskcluster] Artifact "public/my-missing.txt" not found at "/this-file-is-missing.txt"';
```

Comment:
timestamps are included in the prepended text and this test isn't specifically testing that it's added so might as well not check.